docs/custom_mutators.md

```diff
@@ -88,6 +92,11 @@ def queue_new_entry(filename_new_queue, filename_orig_queue):
     This method determines whether the custom fuzzer should fuzz the current
     queue entry or not
 
+- `fuzz_count` (optional):
+
+    This method can be used to instruct afl-fuzz how often to perform a fuzz
```

Comment:
I don't really get this and I doubt the users would: what is a fuzzing attempt? what is 'this input data'? Maybe we should give an example?

Reply:
better?